inc/components/class-image.php

```diff
@@ -377,13 +379,13 @@ public function get_lqip_src() {
 
 		// Return early if no aspect ratio is set.
 		if ( empty( $aspect_ratio ) ) {
-			return $this;
+			return '';
```

Comment:
If you type-hinted this function, per comment above you'd want to update this to be `null`
```suggestion
			return null;
```